### DISCUSSION TITLE
Nextjs static site build

### DISCUSSION THREAD

**Discussion by SamTolmay** [lowdefy/lowdefy#1673] [Ideas and feature requests]
See #537

Currently, a Lowdefy server is required to host a Lowdefy app.

It would be possible to host a Lowdefy app as a static site, that could be served as an S3 website, Cloudflare pages, with nginx etc.

This would mean the app would not be able to use features like user authentication, connections and secrets.

To do this will require the page configuration query in the renderer to be changed to fetch a static JSON file, instead of querying the server, and therefore it makes sense to do this in version 4 when GraphQL has been replaced by a REST API.

**Comment by mueckinger** [comment]
Hey,

is this now possible in Lowdefy 4 as mentioned above?
